admin/install.php

```diff
@@ -292,7 +292,7 @@ function print_test( $p_test_description, $p_result, $p_hard_fail = true, $p_mes
 <?php
 	print_test(
 		'Checking PHP version (your version is ' . phpversion() . ')',
-		check_php_version( phpversion() ),
+		version_compare( phpversion(), PHP_MIN_VERSION, '>=' ),
 		true,
 		'Upgrade to a more recent version of PHP'
```

Comment:
Maybe change this message to upgrade to a version equal to or newer than min version.

Reply:
This code can't be reached at all, as I introduced a check in an early state in core.php some while ago to prevent using non-supported old PHP versions, see https://github.com/mantisbt/mantisbt/commit/1eb5ff1c7e1d8d99e97eadbd54da087d38b0fa55

What you will get is a good understandable message, e. g.  something like
```
FATAL ERROR: Your version of PHP is too old. MantisBT requires 7.0.0 or newer
Your are running PHP version 5.6.40
```

I pushed one more commit to remove the unreachable code.